README.md

```diff
@@ -42,9 +42,10 @@ This plugin currently supports the following SonarQube [metrics](https://docs.so
 - Percentage of line coverage - [`line_coverage`](https://docs.sonarqube.org/display/SONAR/Metric+Definitions#MetricDefinitions-Tests)
 
 In addition to the above, the plugin reports the following custom coverage metrics:
```

Comment:
I think, we should clarify that these metrics are extracted from the scoverage report, and are not computed (except for aggregating by filename).

Reply:
Is that what you were after? (see d593721)